auth/api/grpc/endpoint.go

```diff
@@ -62,17 +62,26 @@ func authorizeEndpoint(svc auth.Service) endpoint.Endpoint {
 			return authorizeRes{}, err
 		}
 
-		_, err := svc.Identify(ctx, req.token)
+		authorized, err := svc.Authorize(ctx, req.Sub, req.Obj, req.Act)
```

Comment:
Can we pass AuthReq here, as everywhere else?

Reply:
Yes, we can pass `AuthReq` but it causes `import cycle not allowed` since `"github.com/mainflux/mainflux/auth"` and [`"github.com/mainflux/mainflux/auth/api/grpc"`](https://github.com/buraksekili/mainflux/blob/ee866f1fc7dd5a2bfa54651f1e6a0729d2ca8ea5/auth/api/grpc/requests.go#L7) call each other after importing `AuthReq`. What if we create a new generic struct (inside of `auth/policies.go`) instead of passing 3 strings?

Reply:
Yea new local struct which will wrap req params sounds great.